Library/Homebrew/formula.rb

```diff
@@ -1622,6 +1622,19 @@ def std_meson_args
     ["--prefix=#{prefix}", "--libdir=#{lib}", "--buildtype=release", "--wrap-mode=nofallback"]
   end
 
+  # Standard parameters for pip builds.
+  sig {
+    params(prefix:          T.nilable(T.any(String, Pathname)),
+           build_isolation: T::Boolean).returns(T::Array[String])
+  }
+  def std_pip_args(prefix: self.prefix, build_isolation: false)
+    args = ["--verbose", "--no-deps", "--no-binary=:all:", "--ignore-installed",
+            "--use-feature=no-binary-enable-wheel-cache"]
+    args << "--prefix=#{prefix}" if prefix.present?
```

Comment:
This will probably also work, unless you prefer a kwarg.
```suggestion
  def std_pip_args(install_prefix = prefix, build_isolation: false)
    args = ["--verbose", "--no-deps", "--no-binary=:all:", "--ignore-installed",
            "--use-feature=no-binary-enable-wheel-cache"]
    args << "--prefix=#{install_prefix}" if install_prefix.present?
```

Reply:
Let's prefer kwargs, yeh.

Reply:
I *slightly* lean towards a kwarg, mainly for readability and consistency with the other `std_*_args` functions here. but no strong opinion either way